src/Reexport.jl

```diff
@@ -14,7 +14,7 @@ function reexport(m::Module, ex::Expr)
         return Expr(:block, map(e -> reexport(m, e), ex.args)...)
     end
 
-    ex.head in (:module, :using, :import) ||
+    ex.head isa Symbol && ex.head::Symbol in (:module, :using, :import) ||
```

Comment:
Why the check for whether the head is a symbol? That's guaranteed to be the case since `ex` is an `Expr` and the `head` field of `Expr` objects is typed as `Symbol`.

Reply:
You're right, the first check is unnecessary, the type assertion is just to help inference see that the method being called is `in(::Symbol, ::Tuple{Vararg{Symbol}})` so it infers it correctly.

Reply:
You're right, the first check is unnecessary, the type assertion is just to help inference see that the method being called is `in(::Symbol, ::Tuple{Vararg{Symbol}})` so it infers it correctly.